apps/src/code-studio/assets/SpriteUpload.jsx

```diff
@@ -0,0 +1,189 @@
+import React from 'react';
+import color from '@cdo/apps/util/color';
+import {makeEnum} from '@cdo/apps/utils';
+import {getManifest} from '@cdo/apps/assetManagement/animationLibraryApi';
+
+const SpriteLocation = makeEnum('library', 'level');
+
+export default class SpriteUpload extends React.Component {
```

Comment:
i'm now realizing that this component is tightly tied to spritelab, which is okay for now, but we would need to make some small changes for this to work for other labs. we should add a comment above this component explaining that, and we should come back later to make this more generic (even a `lab` prop that can currently only be set to `spritelab` would get us most of the way there!)